java-sdk/src/main/java/org/sagebionetworks/bridge/sdk/UserSession.java

```diff
@@ -56,33 +66,24 @@ public boolean isConsented() {
         return ConsentStatus.isUserConsented(consentStatuses);
     }
     
-    public SharingScope getSharingScope() {
-        return this.sharingScope;
+    public StudyParticipant getStudyParticipant() {
+        return this.studyParticipant;
     }
     
     public boolean hasSignedMostRecentConsent() {
         return ConsentStatus.isConsentCurrent(consentStatuses);
     }
     
-    public Set<String> getDataGroups() {
-        return dataGroups;
-    }
-    
-    public String getId() {
-        return id;
-    }
```

Comment:
We expect the session to still contain the ID (see BridgeAdminClient). Why not make it part of the UserSession object?